src/pointers.cpp

```diff
@@ -256,7 +256,7 @@ namespace big
         // Read Bitbuffer Boolean
         {
             "RBB",
-            "E8 ? ? ? ? 84 C0 74 41 48 8D 56 2C",
+            "E8 ? ? ? ? 84 C0 74 ? 48 8D 56 ? 44 8D 43 ? 48 8B CF E8 ? ? ? ? 84 C0 74 ? 48 8D 56 ? 44 8D 43 ? 48 8B CF E8 ? ? ? ? 84 C0 74 ? 48 8D 56 ? 44 8D 43 ? 48 8B CF E8 ? ? ? ? 84 C0 74 ? BB",
```

Comment:
This signature should be updated to be shorter, it won't degrade the speed of the scanner much however it's most likely better to use a shorter signature that also points to it.

Be it from a different part of the assembly, adding, ripping; differently.